account_financial_report/view/report_template.xml

```diff
@@ -6,7 +6,7 @@
         <xpath expr="." position="inside">
             <link href="/account_financial_report/static/src/css/report.css" rel="stylesheet"/>
             <script type="text/javascript"
-                    src="/account_financial_report/static/src/js/account_financial_report_backend.js"/>        
+                    src="/account_financial_report/static/src/js/account_financial_report_backend.js"/>
             <script type="text/javascript"
                     src="/account_financial_report/static/src/js/account_financial_report_widgets.js"/>
```

Comment:
This is the JS file I mean in the previous comment, it contains the listeners for the click actions.